Description: using the debian python shebang line
---

Index: git-multimail/git-multimail/git_multimail.py
===================================================================
--- git-multimail.orig/git-multimail/git_multimail.py	2015-05-23 14:29:30.000000000 +0200
+++ git-multimail/git-multimail/git_multimail.py	2015-05-23 14:29:30.000000000 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python2
+#!/usr/bin/python
 
 # Copyright (c) 2012-2014 Michael Haggerty and others
 # Derived from contrib/hooks/post-receive-email, which is
Index: git-multimail/git-multimail/migrate-mailhook-config
===================================================================
--- git-multimail.orig/git-multimail/migrate-mailhook-config	2015-05-23 14:29:30.000000000 +0200
+++ git-multimail/git-multimail/migrate-mailhook-config	2015-05-23 14:29:30.000000000 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python2
+#!/usr/bin/python
 
 """Migrate a post-receive-email configuration to be usable with git_multimail.py.
 
Index: git-multimail/git-multimail/post-receive
===================================================================
--- git-multimail.orig/git-multimail/post-receive	2015-05-23 14:29:30.000000000 +0200
+++ git-multimail/git-multimail/post-receive	2015-05-23 14:33:27.525129162 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python2
+#!/usr/bin/python
 
 """Example post-receive hook based on git-multimail.
 
